src/pages/Signup.jsx

```diff
@@ -101,83 +136,74 @@ const SignupScreen = () => {
-              zIndex: 1,
-            }}
-          >
-            {address ? (
-              <img
-                src={URL.createObjectURL(image)}
-                alt="profile pic"
-                style={{
-                  width: "100px",
-                  height: "100px",
-                  borderRadius: "100%",
-                }}
-              />
-            ) : (
-              <div style={{ marginTop: "30px" }}>PROFILE PICTURE</div>
-            )}
-          </div>
-          <Input
+          <label htmlFor="file">
+            <div className="img-outer">
+              {address ? (
+                <img
+                  src={URL.createObjectURL(image)}
+                  alt="profile pic"
+                  className="img-inner"
+                />
+              ) : (
+                <div className="img-inner">Profile Picture</div>
+              )}
+            </div>
+          </label>
+
+          <input
             type="text"
-            placeholder="USERNAME"
-            required
+            placeholder="Username"
             value={username}
-            style={{ margin: "5%", color: "var(--color)" }}
             onChange={(e) => setUsername(e.target.value)}
           />
-          <Input
-            type="text"
-            placeholder="EMAIL"
+          <input
+            type="email"
+            placeholder=" Email"
             value={email}
-            style={{ margin: "5%", color: "var(--color)" }}
             onChange={(e) => setEmail(e.target.value)}
           />
-          <Input
+          <input
             type="password"
-            placeholder="PASSWORD"
+            placeholder=" Password"
             value={password}
-            style={{ margin: "5%", color: "var(--color)" }}
             onChange={(e) => setPassword(e.target.value)}
           />
-          <div className="file-input">
-            <input
-              type="file"
-              id="file"
-              className="file"
-              onChange={handleChange}
-              accept="image/*"
-            />
-            <label htmlFor="file">Select Profile Picture</label>
+          <button type="submit" className="button signup">
+            Sign Up <FontAwesomeIcon icon={faRightToBracket} />
+          </button>
+          <div className="or">
+            <div className="line" />
+            <div style={{ padding: "9px" }}>or</div>
+            <div className="line" />
+          </div>
+          <div className="google-fb-login">
+            <button className="button" type="submit" onClick={signInWithGoogle}>
+              <FontAwesomeIcon icon={faGoogle} />
+            </button>
+            <button
+              className="button"
+              type="submit"
+              onClick={signInWithFacebook}
+            >
+              <FontAwesomeIcon icon={faSquareFacebook} />
+            </button>
           </div>
-          <AnimatedButton
+          <button
```

Comment:
login button here should not be of type submit and navigate to dummygram/login instead